tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -6,24 +6,24 @@ summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access
 # Configure Amazon S3 Access and GCS Access
 
 If your source data is stored in Amazon S3 or GCS buckets, before importing or migrating the data to TiDB Cloud, you need to configure cross-account access to the buckets. This document describes how to do this.
-
-- [Configure Amazon S3 access](#configure-amazon-s3-access)
-- [Configure GCS access](#configure-gcs-access)
+  - [Configure Amazon S3 access](#configure-amazon-s3-access)
+  - [Configure GCS access](#configure-gcs-access)
 
 ## Configure Amazon S3 access
 
 To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the following steps to configure the bucket access for TiDB Cloud and get the Role-ARN. Once the configuration is done for one TiDB cluster in a project, all TiDB clusters in that project can use the same Role-ARN to access your Amazon S3 bucket.
```

Comment:
“Once the configuration is done for one TiDB cluster in a project, all TiDB clusters in that project can use the same Role-ARN to access your Amazon S3 bucket.” The premise of this sentence is  only when the role is created without  “Require external ID” . IF the role was created with account id and external id, only the correspond TiDB cluster could access the bucket.

Reply:
Revised as suggested.